### PR TITLE
Added tool to extract glbs from a cmpt

### PR DESCRIPTION
**Pull request by lilleyse**
I forgot I had this lying around - should be ready to review.

### CODE REVIEW

File: tools/bin/3d-tiles-tools.js [hunk 1]
```diff
@@ -265,6 +265,53 @@ function readB3dmWriteGlb(inputPath, outputPath, force) {
...
+
+function readCmptWriteGlb(inputPath, outputPath, force) {
+    outputPath = defaultValue(outputPath, inputPath.slice(0, inputPath.length - 5));
+    return fsReadFile(inputPath)
+        .then(function(data) {
+            var tiles = extractCmpt(data);
+            var tilesLength = tiles.length;
+            var tilePaths = new Array(tilesLength);
+            if (tilesLength === 0) {
+                console.log('No glbs found in this cmpt tile.');
```
Comment (pjcozzi) [inline tools/bin/3d-tiles-tools.js]: "this?"

Why not output the path?

**Comment by pjcozzi** [comment]
@lilleyse when I first saw this, I thought we should change this to be a stage that extracts tiles from composites, that is then piped into another stage that extracts glbs from tiles.

What do you think?

What you have here could be fine for today's needs (debugging glbs) and we could just close #39?

**Comment by pjcozzi** [comment]
Code and tests are good.

**Comment by lilleyse** [comment]
> @lilleyse when I first saw this, I thought we should change this to be a stage that extracts tiles from composites, that is then piped into another stage that extracts glbs from tiles.

I think that's a good idea and better separation of tasks. I made the changes, but instead of making a `tilesToGlb` stage I just added a helper function in `3d-tiles-tools.js`.

**Pull request merged**